src/Effect.js

```diff
@@ -147,8 +147,15 @@ class Effect extends EventEmitter {
    * @return {number} attribute modified by effect
    */
   modifyAttribute(attrName, currentValue) {
-    const modifier = (this.modifiers.attributes[attrName] || (_ => _)).bind(this);
-    return modifier(currentValue);
+    let modifier = _ => _;
```

Comment:
While you couldn't possibly answer all questions I had, what in the world is let modifier = _ => _ doing?